round4/team-02.md

```diff
@@ -0,0 +1,124 @@
+# Chrome Extension Idea: Video Insight
+
+## Authors
+
+John Guirguis, Jake Brooks, Aditya Maddipatla, Ashwin Balaji, & Zain Khan
+
+## Problem Statement
+
+Many students struggle with understanding the corpus of research we’re provided to read through due to the building nature of STEM fields. As such, it’s impossible to succeed in understanding later areas of work without first grasping prior research (often this research is the product of concepts taught in fragments throughout multiple different courses in undergrad). One of the key pain points we aim to address with this product would be the need to look into and search up terms or concepts that you encounter as you read new material if you haven’t seen it before. Another would be struggling with dense wordy explanations of ideas that could be tailored for a more visual audience. Many of our users would likely be international students with multi-lingual backgrounds, and it might help them to have concepts re-explained in a language they’re more familiar with. Finally, we would hope to address recurring or important explanations (potentially for harder or more useful concepts) with bookmark abilities 
```

Comment:
I think this is a very doable project to do within 4 weeks. I think the biggest challenge you guys wull have to face is personalization/youtube limits for the searches

Reply:
We were planning on making users create their own API key before using the extension so that we are not charged for use of personal API keys. Furthermore, this process will be streamlined through an initial prompt the first time that a user attempts to use the plugin.